src/test/java/bio/terra/service/search/SearchServiceTest.java

```diff
@@ -71,22 +88,48 @@ public void indexSnapshotTest() throws Exception {
         assertEquals(indexName, searchIndexModel.getIndexSummary());
     }
 
+    @Test
+    public void querySnapshotTest() throws Exception {
+        GetAliasesResponse mockResponse = mock(GetAliasesResponse.class);
+        when(mockResponse.getAliases()).thenReturn(Map.of("0f14d0ab-9605-4a62-a9e4-5ed26688389b", Set.of()));
```

Comment:
It would be good to use a named constant or local variable for `"0f14d0ab-9605-4a62-a9e4-5ed26688389b"` so it's clear that the same UUID is used both here and on line 98.